src/oc/debug.cpp

```diff
@@ -153,86 +144,3 @@ void debugzz(Inst* p) {
     }
 #endif /*OCSMALL*/
 }
-
-#if NRN_DIGEST
```

Comment:
I think the `digest*` part was relatively recently added in https://github.com/neuronsimulator/nrn/pull/3135, so while not "used" elsewhere in the codebase, it's not really "unused" either.

Reply:
Of course ;)